dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -396,7 +396,7 @@ alter table t_ds_process_instance drop dependence_schedule_times;
 -- t_ds_task_instance   note: Data migration is not supported
 alter table t_ds_task_instance change process_definition_id task_code bigint(20) NOT NULL COMMENT 'task definition code';
 alter table t_ds_task_instance add task_definition_version int(11) DEFAULT '0' COMMENT 'task definition version' AFTER task_code;
-alter table t_ds_task_instance add task_params text COMMENT 'job custom parameters' AFTER app_link;
+alter table t_ds_task_instance add task_params longtext COMMENT 'job custom parameters' AFTER app_link;
```

Comment:
you need to change the newest sql file. and other database type sql file too.

Reply:
I'll fix it.